hnn_core/drives.py

```diff
@@ -10,14 +10,49 @@
                      _extract_drive_specs_from_hnn_params)
 
 
-def _get_target_populations(weights_ampa, weights_nmda):
+def _get_target_properties(weights_ampa, weights_nmda, synaptic_delays,
+                           location):
+    """Retrieve the the user-defined synaptic weight and delay parameters
+    according to their target populations cell types
+    """
```

Comment:
Docstrings should always be of the format where first there is a one line description, then a blank line, then a longer description. See: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings